openassessment/xblock/test/test_staff_area.py

```diff
@@ -233,6 +235,9 @@ def test_staff_area_student_info_self_only(self, xblock):
         self.assertEquals("Bob Answer 1", context['submission']['answer']['parts'][0]['text'])
         self.assertEquals(None, context['peer_assessments'])
         self.assertEquals("openassessmentblock/staff_area/oa_student_info.html", path)
+        grade_details = context['grade_details']
```

Comment:
Note that I didn't attempt to cover every permutation in unit tests. I am reusing a helper method, and the acceptance test coverage is pretty strong.